.github/workflows/_e2e_tests.yaml

```diff
@@ -72,11 +72,10 @@ jobs:
         name: Set Kind versions
         run: |
           if [ "${{ inputs.all-supported-k8s-versions }}" == "true" ]; then
-            echo "kind-versions=[\"1.22.15\", \"1.23.13\", \"1.24.7\", \"1.25.8\", \"1.26.3\", \"1.27.1\"]" >> $GITHUB_OUTPUT
+            echo "kind-versions=[\"v1.24.14\", \"v1.25.10\", \"v1.26.5\", \"v1.27.2\"]" >> $GITHUB_OUTPUT
```

Comment:
We have not made a decision to drop k8s 1.22 and 1.23 just yet.

Specifically, https://docs.konghq.com/kubernetes-ingress-controller/latest/references/version-compatibility/#general does not have a change proposed to drop any particular k8s versions.

There is prior art w.r.t this, done by @randmonkey at https://docs.google.com/document/d/1Se04bL3JUUSjTbxz8GvyiKA0oPRJzD9NP7UXrTEYZZI/edit#heading=h.hc93t5bgvuoc.

Apart from the doc being pretty descriptive we didn't write down the decision for this.

---

At https://docs.konghq.com/kubernetes-ingress-controller/latest/support-policy/#supported-versions we document "UPSTREAM K8S EOL" but not what version we support. 1.22 has reached EOL October 2022. 1.23 has done so in February 2023.

Maybe it's the time to make this explicit and write it down in user facing docs? 

@mheap @mflendrich thoughts?

Reply:
btw. what is the process to get KIC 2.10. [here](https://docs.konghq.com/kubernetes-ingress-controller/latest/references/version-compatibility/#general)?

Reply:
Make a PR to https://github.com/Kong/docs.konghq.com/ that targets [2.10 branch](https://github.com/Kong/docs.konghq.com/tree/release/kic-2.10)

Reply:
@programmer04 to update this table we should contribute to docs.konghq.com repo, can be a part of this PR: https://github.com/Kong/docs.konghq.com/pull/5645

Reply:
I've made [the commit](https://github.com/Kong/docs.konghq.com/pull/5645/commits/f1b92768b6532dc2e8193172847904e20be229c5), [result is here](https://deploy-preview-5645--kongdocs.netlify.app/kubernetes-ingress-controller/latest/references/version-compatibility/#general).